styles/styles.css

```diff
@@ -678,33 +677,33 @@ button {
 
 .button-container > a.tertiary span.icon svg * {
   stroke: var(--primary-purple);
-  }
+}
 
 /* stylelint-disable-next-line no-descending-specificity */
 .section > div:last-child > *:last-child,
 .section > div:last-child > *:last-child > *:last-child { /* no bottom margin on whatever the last thing + 1 inner is */
   margin-bottom: 0;
   padding-bottom: 0;
-  }
+}
 
 /* stylelint-disable-next-line no-descending-specificity */
 .section > div:first-child > *:first-child,
 .section > div:first-child > *:first-child > *:first-child { /* no top margin on whatever comes next + 1 inner */
   margin-top: 0;
   padding-top: 0;
-  }
+}
 
 .button-container > a.tertiary:hover span.icon svg * {
-  stroke: url("#gradient");
-  }
+  stroke: var(--primary-red);
```

Comment:
Is this according to Figma design? Can https://github.com/hlxsites/merative/issues/164 be closed too then?

Reply:
There isn't actually a spec for hover that includes the arrow, so I made an executive decision to use the last color of the gradient that is on the text link. I think it blends in/has continuity so I'm ready to call this ticket "complete" if you are.
![image](https://user-images.githubusercontent.com/3883795/230377276-64bfe106-705a-49cf-b2e9-5d724c34a34a.png)